projects/canopy/src/lib/forms/toggle/toggle.component.spec.ts

```diff
@@ -64,7 +79,12 @@ class TestToggleComponent {
     });
   }
 
-  isControlInvalid(control: NgControl, form: FormGroupDirective) {
+  isControlInvalid(
+    control: AbstractControl<
+      ɵGetProperty<ɵTypedOrUntyped<never, ɵFormGroupRawValue<never>, never>, 'umbrella'>
```

Comment:
This doesn't look right - pretty sure these types aren't meant for outside consumption

Reply:
fixed